terraform/services/ecs_service/alb.tf

```diff
@@ -12,7 +12,8 @@ resource "aws_alb_target_group" "ecs_service" {
   }
 }
 
-resource "aws_alb_listener_rule" "rule" {
+resource "aws_alb_listener_rule" "path_rule" {
+  count        = "${1 - var.use_host_condition}"
```

Comment:
Yeah, I was looking at using them for the GSIs in DynamoDB, but you can’t stick a `count` on a GSI. 😕